app/assets/javascripts/views/story_view.js

```diff
@@ -301,6 +301,7 @@ module.exports = FormView.extend({
         that.enableForm();
         that.model.set({ editing: editMode });
         that.toggleControlButtons(false);
+        that.highlightStory($(that.storyElementId()));
```

Comment:
For this one, since this is not the one for the search, you can do `that.highlightStory(this.$el)` 😄

Reply:
So, I don't know why but it does not works :( 
When I try to use `that.highlightStory(this.$el);` nothing happens.
When I try to use `that.highlightStory(that.$el);` it works only for old stories.
Could you explain why it happens ?